db/migrate/20161003201337_create_cheeses.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCheeses < ActiveRecord::Migration[5.0]
```

Comment:
Top level class documentation is something you need to address. Either change the .yml or add in comments. Either way works just pick a system and stay consistent :)